File: primatis-web/src/app/notifications/models/notification-unread-count-response.ts
/**
 * Voir `be.primatis.notification.dto.NotificationUnreadCountResponse` côté
 * backend (DEV-DEC-0051). Compteur dédié : jamais dérivé côté frontend à
 * partir de la page courante.
 */
export interface NotificationUnreadCountResponse {
  count: number;
}

File: primatis-api/src/main/java/be/primatis/notification/dto/NotificationUnreadCountResponse.java
package be.primatis.notification.dto;

/**
 * Contrat REST du compteur de Notifications {@code UNREAD} de l'utilisateur
 * authentifié (DEV-10.4, DEV-DEC-0051). DTO dédié plutôt qu'une map
 * anonyme, cohérent avec les conventions backend PRIMATIS (contrats REST
 * explicites, jamais de structure ad hoc).
 */
public record NotificationUnreadCountResponse(long count) {
}
